tests/prompt_test_classification_agent.md

```
@@ -0,0 +1,116 @@
+Every test file **must mirror** its `src/main_app/` counterpart path. Examples:
+
+-   `src/main_app/db/db_Jobs.py` → `tests/unit/main_app/db/test_db_Jobs.py`
+-   `src/main_app/app_routes/auth/routes.py` → `tests/integration/main_app/app_routes/auth/test_routes.py`
+-   `src/main_app/services/jobs_service.py` → `tests/unit/main_app/services/test_jobs_service.py`
+
+## Split File Naming Convention
+
+If a file contains tests of two different types (e.g. unit + integration),
+it must be split into two files using this naming pattern:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)